### PR TITLE
fix(commercetools): reselect shipping delivery

### PR DESCRIPTION
**Pull request by alefbarbeli**
## Description
Changed UI from SfTabs to SfRadio
Delivery selection don't show if C&C not available
Fixed issue where User cannot reselect "Shipping"
Changed naming from "Shipping" to "Delivery"

## Related Issue
Resolves [#OS-147](https://vsf.atlassian.net/browse/OS-147?atlOrigin=eyJpIjoiNTkzOGMzMDQ0NGQyNDNmYmIwZjQ5MTVmNzliZjJlNDUiLCJwIjoiaiJ9)

## Motivation and Context
Bug found on QA test

## How Has This Been Tested?
Ran local dev

## Screenshots:
**BEFORE:**
image
image

**AFTER:**
image
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.

#### Changelog
- [ ] I have updated the Changelog ([V1](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md)) [v2](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and mentioned all breaking changes in the public API.
- [ ] I have documented all new public APIs and made changes to existing docs mentioning the parts I've changed so they're up to date.

#### Tests
- [ ] I have written test cases for my code
- [ ] I have tested my Pull Request on production build and (to my knowledge) it works without any issues
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

> I tested manually my code, and it works well with both:
- [ ] Default Theme
- [ ] Capybara Theme

#### Code standards
- [x] My code follows the code style of this project.

- [x] I have followed [naming conventions](https://github.com/kettanaito/naming-cheatsheet)

#### Docs
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by igorwojciechowski** [comment]
I'm wondering whether it should be merged to `develop` or `release-2.4` - maybe the later would be a better dest branch as it seems to be a fix for a feature we introduced in 2.4? WDYT?

File: packages/commercetools/theme/pages/Product.vue [hunk 1]
```diff
@@ -75,30 +75,45 @@
...
-              title="Click and Collect"
+
+          <div
+            class="product__delivery"
+            v-if="channels.length > 0"
+          >
+            <SfRadio
+              v-e2e="'delivery-option'"
+              name="Delivery"
+              label="Delivery"
```
Comment (dawid-ziobro) [inline packages/commercetools/theme/pages/Product.vue:86]: Should it be hardcoded in english?

File: packages/commercetools/theme/pages/Product.vue [hunk 2]
```diff
@@ -75,30 +75,45 @@
...
+              value="delivery"
+              :selected="selectedDelivery"
+              @input="selectDelivery('delivery')"
+            />
+            <SfRadio
+              v-e2e="'click-collect-option'"
```
Comment (dawid-ziobro) [inline packages/commercetools/theme/pages/Product.vue:92]: You've added `v-e2e` attributes but there's no e2e tests added

File: packages/commercetools/theme/pages/Product.vue [hunk 3]
```diff
@@ -250,8 +266,11 @@ export default {
       const productChannels = product.value?.availability?.channels?.results ?? [];
       return productChannels;
     });
+    const selectedDelivery = ref(null);
+    const selectDelivery = option => selectedDelivery.value = option;
```
Comment (dawid-ziobro) [inline packages/commercetools/theme/pages/Product.vue:270]: To increase readability maybe change this function name to `setSelectedDelivery`, what do you think?

File: packages/commercetools/theme/pages/Product.vue [hunk 4]
```diff
@@ -83,25 +83,25 @@
             <SfRadio
               v-e2e="'delivery-option'"
               name="Delivery"
-              label="Delivery"
+              :label="$t('Delivery')"
```
Comment (dawid-ziobro) [inline packages/commercetools/theme/pages/Product.vue:86]: Please add translations for all new phrases

File: packages/commercetools/theme/pages/Product.vue (hunk 3 above)
Comment (alefbarbeli) [reply packages/commercetools/theme/pages/Product.vue]: done!

**Review by dawid-ziobro — APPROVED** [review]

**Pull request merged**